### PR TITLE
Add Headwind MDM

### PR DESCRIPTION
**Pull request by okhosting**

### CODE REVIEW

**Comment by okhosting** [comment]
Add mobile-toolkit

File: README.md [hunk 1]
```diff
@@ -101,4 +102,6 @@ Applicable to all platforms.
 
 ## Tools
 
-  * [micromdm/scep](https://github.com/micromdm/scep) - A SCEP server written in Go
\ No newline at end of file
+  * [micromdm/scep](https://github.com/micromdm/scep) - A SCEP server written in Go
+  * [IntergalacticPenguin/mobile-toolkit](https://github.com/IntergalacticPenguin/mobile-toolkit) - Command line tool for Android & iOS device management
+  * [Ylianst/MeshCentral](https://github.com/Ylianst/MeshCentral) - With MeshCentral, you can run your own web server and it to remotely manage and control computers on a local network or anywhere on the internet.
```
Comment (petarov) [inline README.md]: `...and it to remotely` - seems a verb is missing here?

**Comment by okhosting** [comment]
@petarov you are correct, here is the fix

**Comment by petarov** [comment]
Looks good. Thanks for adding these. 🍻

**Pull request merged**